applications/osb-portal/src/store/actions/workspaces.ts

```diff
@@ -1,6 +1,6 @@
 import { WorkspaceActions } from '../reducers/workspaces'
 
-export const { selectWorkspace, refreshWorkspace, refreshWorkspaces, deleteWorkspace, updateWorkspace, resourceAdded } = WorkspaceActions;
+export const { selectWorkspace, refreshWorkspace, refreshWorkspaces, deleteWorkspace, updateWorkspace, resourceAdded, cloneWorkspace } = WorkspaceActions;
```

Comment:
as this was added, WorkspaceInteraction will have the cloneWorkspace prop added without passing it explicitly